sdk/tdf_test.go

```diff
@@ -612,9 +679,13 @@ func (s *TDFSuite) Test_TDFWithAssertion() {
 				s.Require().NoError(err)
 			}(fileWriter)
 
-			tdfObj, err := s.sdk.CreateTDF(fileWriter, bufReader,
-				WithKasInformation(kasURLs...),
-				WithAssertions(test.assertions...))
+			createOptions := []TDFOption{WithKasInformation(kasURLs...),
+				WithAssertions(test.assertions...)}
+			if test.useHex {
+				createOptions = append(createOptions, WithTargetMode("0.0.0"))
+			}
+
+			tdfObj, err := s.sdk.CreateTDF(fileWriter, bufReader, createOptions...)
```

Comment:
Can we add asserting that the version is what we expect and also that the a segment and rootSignature is hex encoded.

Reply:
updated